src/components/App.js

```diff
@@ -371,7 +385,10 @@ export default class App extends React.Component {
       )
     );
 
-  onPlayerError = () => {
+  onPlayerError = err => {
+    // NOTE: this might be a bit noisy, so we should monitor it before promoting
+    // coderadio
+    Sentry.captureException(err);
     /*
      * This error handler works as follows:
      * - When the player cannot play the url:
```

Comment:
I plan on leaving this running locally for a while so if this handler starts exploding I'll let you know. 🙂

Reply:
so the funny thing with that error is that the player does not error out :)

Reply:
Right so it wouldn't catch here, but it should catch in the `NchanSubscriber` handler right?

https://github.com/freeCodeCamp/coderadio-client/blob/f39b74e03b1e6184fc9ebfc56b5db0ac1a1a3e72/src/components/App.js#L16-L22